sdk/avalon_sdk/ethereum/proxy/ethereum_worker_registry.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2020 Intel Corporation
```

Comment:
Is this file newly created? We already have ethereum_worker_registry.py in ethereum, you can use that righ?